e2e/android/app/src/main/java/com/e2eoktareactnative/MainApplication.kt

```diff
@@ -1,63 +1,44 @@
 package com.e2eoktareactnative
```

Comment:
I changed this file based on changes suggested by [react-native upgrade helper](https://react-native-community.github.io/upgrade-helper/) for react-native 0.73 -> 0.74